### DISCUSSION TITLE
Fijar imagen con coordenadas de otra

### DISCUSSION THREAD

**Discussion by KevinVillanuev4** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#437] [Q&A]
EStoy intentando hacer mi pokedex pero quiero ocupar una imagen de fondo y apartir de ella acomodar divs que siempre esten en esa posicion para mostrar la info, companeros con experiencia en css o alguna otra herramienta eso es posible?
![image](https://user-images.githubusercontent.com/99345183/158739526-3eaf35e6-ac27-4063-9d72-6fd842d0f7fb.png)
asi se ve en grande pero cuando la hago pequena se ve asi
![image](https://user-images.githubusercontent.com/99345183/158739677-bac2e830-a3ed-460c-9ad7-45c46e9a32d1.png)

**Comment by Ingrzs** [comment]
Si es posible, al div tendrás que irle calculando la medida de ancho y altura con px y acada div dale una position relative ya lo mueves con top -left-rigt-bottom (usando valores con %) segun sea el caso.
Cuando la haces pequeña para eso usa el mediaquery (responsive)y recalculas el tamaño y listo 
 eso seria todo.
